coral/src/app/features/configuration/clusters/components/ClustersTable.tsx

```diff
@@ -123,14 +139,41 @@ const ClustersTable = (props: ClustersTableProps) => {
         serviceName: cluster.serviceName,
         projectName: cluster.projectName,
       },
+      canDeleteCluster: cluster.showDeleteCluster,
     };
   });
 
   if (rows.length === 0) {
     return <EmptyState title="No Clusters">No clusters found.</EmptyState>;
   }
 
-  return (
+  return isAdminUser ? (
```

Comment:
This ternary was added because types do not allow `| undefined` for `menu` and `onAction`

<img width="1495" alt="Screenshot 2024-03-15 at 13 23 18" src="https://github.com/Aiven-Open/klaw/assets/20607294/56df3923-d1e8-438b-ae40-87f7fefbfd68">

Reply:
Thanks for the explanation!